ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -504,6 +504,14 @@ func (s *Service) buildCreateServiceInput(serviceName, taskDefName string, desir
 	if s.healthCheckGP != nil && s.loadBalancer == nil {
 		return nil, fmt.Errorf("--%v is only valid for services configured to use load balancers", flags.HealthCheckGracePeriodFlag)
 	}
+	// TODO: revert to "LATEST" when latest refers to 1.4.0
+	platformVersion := aws.String("LATEST")
+	if len(ecsParams.TaskDefinition.EFSVolumes) != 0 {
+		log.Warnf("Detected an EFS Volume in task definition %s", taskDefName)
+		log.Warn("EFS requires Fargate platform version 1.4.0, which includes changes to the networking flows for VPC endpoint customers.")
```

Comment:
not clear from this statement alone that we're *using* 1.4.0 now, just that it's required. Since it comes right after the msg re: EFS volume detected, does it make sense in context to say "Using required Fargate platform version 1.4.0, which..." ?

Reply:
Yes, it does!